src/app/shared/components/incentive-dialog/incentive-dialog.component.html

```diff
@@ -0,0 +1,56 @@
+<div class="mat-dialog-image">
+  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48">
+    <defs>
+      <style>
+        .cls-1 {
+          fill: #424752;
+        }
+
+        .cls-2 {
+          fill: #e6e6e6;
+        }
+
+        .cls-3 {
+          fill: #6a7894;
+        }
+
+        .cls-4 {
+          fill: #2f333b;
+        }
+
+        .cls-5 {
+          fill: #3f434d;
+        }
+
+        .cls-6 {
+          fill: #ff7360;
+        }
+
+        .cls-7 {
+          fill: #333843;
+        }
+      </style>
+    </defs>
+    <g id="clock" transform="translate(-256 -51)">
+      <circle id="Эллипс_12" data-name="Эллипс 12" class="cls-1" cx="24" cy="24" r="24" transform="translate(256 51)"/>
```

Comment:
Do we need `id` and `data-name` attributes here? If yes, could you please name it in English?